examples/Application/Traffic-Light-Control/agent/agent.py

```diff
@@ -0,0 +1,64 @@
+#   Copyright (c) 2021 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import paddle
+import parl
+import numpy as np
+
+
+class Agent(parl.Agent):
+    def __init__(self, algorithm, config):
+        super(Agent, self).__init__(algorithm)
+
+        self.config = config
+        self.epsilon = self.config['epsilon']
+
+    def sample(self, obs):
+        # The epsilon-greedy action selector.
+        def sample_random(act_dim):
```

Comment:
Please remove the simple function. Just call `np.random.randint(0, act_dim)`.

Reply:
fixed